test/verify/check-openshift

```diff
@@ -608,6 +608,47 @@ class TestRegistry(MachineCase):
         b.click(".btn-primary")
         b.wait_not_present("modal-dialog")
 
+        # try to add user with invalid name from testprojectuserproj page
+        b.go("#/projects/testprojectuserproj")
+        b.click("a i.pficon-add-circle-o")
```

Comment:
Nice catch, thanks! I never saw a local nor production failure on that one, but it can't hurt, so pushed.